index.bs

```diff
@@ -262,7 +267,16 @@ EventData = (
 
 ## Session ## {#session}
 
-WebDriver BiDi uses the same [=/session=] concept as WebDriver.
+WebDriver BiDi extends the [=/session=] concept from WebDriver.
+
+A [=/session=] has a <dfn>BiDi flag</dfn>, which is false unless otherwise
+stated.
+
+A <dfn>WebDriver BiDi Session</dfn> is a [=/session=] which has the [=BiDi flag=]
```

Comment:
Since it's just "session" in https://w3c.github.io/webdriver/#dfn-sessions I figure the more consistent name would be just "BiDi session" without "WebDriver":

```suggestion
A <dfn>BiDi session</dfn> is a [=/session=] which has the [=BiDi flag=]
```